flowers_market/urls.py

```diff
@@ -13,9 +13,10 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.contrib import admin
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^flowers/', include('site_flowers.urls'))
 ]
```

Comment:
add to .gitignore format file .jpg
please don't use git add .
add by one file step by step
git add file_name1 file_name2